sites/docs/content/index.md

```diff
@@ -5,3 +5,51 @@ description: Runes are magic, but what good is magic if you don't have a wand?
 
 Runed is a collection of utilities for Svelte 5 that make composing powerful applications and
 libraries a breeze, leveraging the power of [Svelte Runes](https://svelte.dev/blog/runes).
+
+## Why Runed?
+
+Svelte 5 Runes unlock immense power by providing a set of primitives that allow us to build
+impressive applications and libraries with ease. However, building complex applications often
+requires more than just the primitives provided by Svelte Runes.
+
+Runed takes those primitives to the next level by providing:
+
+- **Powerful Utilities**: A set of carefully crafted utility functions and classes that simplify
+  common tasks and reduce boilerplate.
+- **Collective Efforts**: We often find ourselves writing the same utility functions over and over
+  again. Runed aims to provide a single source of truth for these utilities, allowing the community
+  to contribute, test, and benefit from them.
+- **Consistency**: We aim to provide consistent APIs and behaviors across all utilities, so you can
+  focus on building your projects instead of constantly learning new APIs.
```

Comment:
```suggestion
- **Consistency**: A consistent set of APIs and behaviors across all utilities, so you can
  focus on building your projects instead of constantly learning new APIs.
```